particles/Tutorial/Kotlin/4_Handles/GetPerson.kt

```diff
@@ -0,0 +1,30 @@
+package arcs.tutorials
+
+import arcs.*
+import kotlin.native.internal.ExportForCppRuntime
+
+
+/**
+ * Sample WASM Particle.
+ */
+class GetPersonParticle : Particle() {
+    private val person = Singleton { GetPerson_Person() }
+
+    override fun getTemplate(slotName: String) = """
+        <input placeholder="Enter your name" spellcheck="false" on-change="onNameInputChange">
+        <div slotid="greetingSlot"></div>""".trimIndent()
+
+    init {
+      registerHandle("person", person)
+
+      eventHandler("onNameInputChange") {
+        val p = person.get() ?: GetPerson_Person()
+        p.name = it["value"] ?: "Human"
```

Comment:
For normal dev, this is totally fine to do... but since this is a tutorial, it might be helpful to give the `it` param a more readable/understandable name for the tutorial-followers to grasp.